0002-legate-ofi.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/install.py b/install.py
+index 25bf4e7..38f9cec 100755
+--- a/install.py
++++ b/install.py
+@@ -452,7 +452,7 @@ def install(
+     if ucx_dir:
+         cmake_flags += [f"-DUCX_ROOT={ucx_dir}"]
+     if conduit:
+-        cmake_flags += [f"-DGASNet_CONDUIT={conduit}"]
++        cmake_flags += [f"-DGASNet_CONDUIT={conduit}"] #.split('-')[0]}"]
+     if cuda_dir:
+         cmake_flags += [f"-DCUDAToolkit_ROOT={cuda_dir}"]
+     if thrust_dir:
+@@ -643,7 +643,7 @@ def driver():
+         # TODO: To support UDP conduit, we would need to add a special case on
+         # the legate launcher.
+         # See https://github.com/nv-legate/legate.core/issues/294.
+-        choices=["ibv", "ucx", "aries", "mpi"],
++        choices=["ibv", "ucx", "aries", "mpi", "udp", "ofi-slingshot11"],
```

Comment:
Let's skip "udp", we know it needs some changes to the driver before it can work out of the box, see https://github.com/nv-legate/legate.core/issues/294.

Reply:
Hrmm that commit looks a bit askance in isolation, but I think it is just because it is a diff of a diff